csharp/src/Glacier2/Glacier2.csproj

```diff
@@ -3,13 +3,12 @@
   <Import Project="../../msbuild/ice.common.props" />
   <PropertyGroup>
     <AssemblyName>Glacier2</AssemblyName>
+    <Description>Client library for the Glacier2 service</Description>
```

Comment:
I used "Client library" even though it's more than a client library.

After researching how to call this assembly, "client library" appears to be to most common / understandable description.